### PR TITLE
Issue 1572 Modus-Table: Fix when reordering the columns, the cell text is getting highlighted

### PR DESCRIPTION
**Pull request by rjavier-trimbler**
## Description
Including user-select CSS property to controls whether the user can't select when column reorder is enable

References #1572 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: stencil-workspace/src/components/modus-table/parts/columnHeader/modus-table-column-header.tsx [hunk 1]
```diff
@@ -59,6 +59,12 @@ export const ModusTableColumnHeader: FunctionalComponent<ModusTableColumnHeaderP
       `}
       style={{
         width: `${getSize()}px`,
+       ...columnReorder && ({
+        'user-select': 'none',
+        '-webkit-user-select': 'none',
+        '-khtml-user-select': 'none',
```
Comment (coliff) [inline stencil-workspace/src/components/modus-table/parts/columnHeader/modus-table-column-header.tsx:65]: minor thing, but no need for `-khtml-user-select`.
REF:
- https://caniuse.com/mdn-css_properties_user-select
- https://autoprefixer.github.io/

Comment (rjavier-trimbler) [reply stencil-workspace/src/components/modus-table/parts/columnHeader/modus-table-column-header.tsx]: Removed!

**Review by coliff — DISMISSED** [review]

File: stencil-workspace/src/components/modus-table/parts/columnHeader/modus-table-column-header.tsx [hunk 2]
```diff
@@ -59,6 +59,11 @@ export const ModusTableColumnHeader: FunctionalComponent<ModusTableColumnHeaderP
       `}
       style={{
         width: `${getSize()}px`,
+       ...columnReorder && ({
```
Comment (msankaran0712) [inline stencil-workspace/src/components/modus-table/parts/columnHeader/modus-table-column-header.tsx:62]: Can you please check whether it is possible to add the styles to an existing class in modus-table.scss ?

Comment (rjavier-trimbler) [reply stencil-workspace/src/components/modus-table/parts/columnHeader/modus-table-column-header.tsx]: done!

**Review by msankaran0712 — APPROVED** [review]

**Pull request merged**